### PR TITLE
SC-4577 - datenschutz per school

### PR DESCRIPTION
**Pull request by pkleybolte**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-4577
https://github.com/schul-cloud/schulcloud-client/pull/1911

## Screenshots of UI changes

image

image

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: src/store/terms-and-conditions.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// import mergeDeep from "@utils/merge-deep";
+// import serviceTemplate from "@utils/service-template";
+
+// const base = serviceTemplate("terms-and-conditions");
```
Comment (BlackGhostHD) [inline src/store/terms-and-conditions.js:4]:
```suggestion
```

File: src/store/terms-and-conditions.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const module = {
+	state: () => ({ consentVersion: null }),
+	actions: {
+		schoolTermsPresent: async function ({ commit }, schoolId) {
```
Comment (BlackGhostHD) [inline src/store/terms-and-conditions.js:9]:
```suggestion
		async schoolTermsPresent({ commit }, schoolId) {
```

Comment (BlackGhostHD) [reply src/store/terms-and-conditions.js:9]: may as well stay that way

**Comment by BlackGhostHD** [comment]
@schul-cloud-bot update screenshots

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

**Comment by CordlessWool** [comment]
@schul-cloud-bot update screenshots

**Comment by luizzaly** [comment]
@schul-cloud-bot update screenshots

**Pull request merged**